### PR TITLE
chore: Update to .7

### PR DESCRIPTION
**Pull request by ryan-s-roberts**
* Use registry version BTP2.1.0-0.7.1
* Support the new version embedding by echoing from make
* Clean up .dockerignore to ignore all + allow
* Add log filters to calm down the wasm tracing

Ref: CHRON-381

Note this will not build until there is a published BTP2.1.0-0.7.1

# PR Checklist

## Please complete this checklist after opening your PR

* [x] I have updated documentation as necessary
* [x] I have updated helm chart(s) if needed
* [x] I have added tests if needed
* [ ] All new and existing tests are passing
* [x] Any breaking changes are clearly flagged and documented
* [x] I’ve included a link to any relevant ticket(s)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,6 +42,7 @@ define domain_tmpl =
 .PHONY: all-domains
 all-domains: $(1)
 
+
```
Comment (mtbc) [inline Makefile:45]: intended?

File: docker/chronicle.dockerfile [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # syntax=docker/dockerfile:1.4
 ARG CHRONICLE_BUILDER_IMAGE=blockchaintp/chronicle-builder-${TARGETARCH}
-ARG CHRONICLE_VERSION=BTP2.1.0-0.6.2
+ARG CHRONICLE_VERSION=BTP2.1.0-0.7.2
 
 FROM ${CHRONICLE_BUILDER_IMAGE}:${CHRONICLE_VERSION} as cache
 ARG RELEASE=no
 ARG FEATURES=""
 
+COPY .VERSION /app/.VERSION
```
Comment (scealiontach) [inline docker/chronicle.dockerfile:9]: this file doesn't exist, nor is it generated by the Makefile

**Review by scealiontach — CHANGES_REQUESTED** [review]

File: docker/chronicle.dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1.4
 ARG CHRONICLE_BUILDER_IMAGE=blockchaintp/chronicle-builder-${TARGETARCH}
-ARG CHRONICLE_VERSION=BTP2.1.0-0.6.2
+ARG CHRONICLE_VERSION=BTP2.1.0-0.7.2
```
Comment (scealiontach) [inline docker/chronicle.dockerfile:3]: update this one to 0.7.3 as well

**Review by scealiontach — CHANGES_REQUESTED** [review]

**Review by scealiontach — APPROVED** [review]

File: policies/common_rules.rego [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import input
+
+allowed := {"chronicle", "anonymous"}
+
+allowed_users {
+  input.type in allowed
+}
+
+
+
```
Comment (mtbc) [inline policies/common_rules.rego:13]: can omit the trailing blanks

**Comment by mtbc** [comment]
`make run-stl-artworld` gives me an artworld that interacts fine via GraphQL.

**Pull request merged**